core/deployment/src/test/java/io/quarkiverse/langchain4j/test/guardrails/OutputGuardrailValidationTest.java

```diff
@@ -91,14 +92,6 @@ void testFatalException() {
         assertThat(fatal.spy()).isEqualTo(1);
     }
 
-    @Test
-    @ActivateRequestContext
-    void testRepromptingWithoutRetry() {
```

Comment:
The new API makes impossible to ask for a reprompt without a retry, so I simply removed this test.

Reply:
+1 - I was very unhappy about that.